File: 1 - Sequenciais/sequenciais5.c
//Faça um programa que converta metros para centímetros.

#include <stdio.h>

int main()
{
  printf("Digite um valor, em metros: \n");
  int metros;
  scanf("%d", &metros);
  printf("O valor, em centimetros, eh de: %d", metros * 100);
  return 0;
}

File: 1 - Sequenciais/Java/sequenciais5.java
//Faça um programa que converta metros para centímetros

import java.util.Scanner;

public class sequenciais5 {

  public static void main(String[] args) {
    System.out.println("Digite um valor, em metros: ");
    int metros = new Scanner(System.in).nextInt();
    System.out.println("O valor, em centimetros, eh de: " + (metros * 100));
  }

}
